test/basic.js

```diff
@@ -15,6 +15,11 @@ describe('Basic', () => {
     }, 1000);
   });
 
+  it('should only test in the correct environment', function(done) {
+      this.skip();
+      done();
```

Comment:
Since it is not an asynchronous test, there is no need to call `done()` here.

Reply:
a6605c8c8c2071bf25b46ec83602b2932c6360e7
delete `done()`